keyboards/keebio/nyquist/keymaps/revkev/keymap.c

```diff
@@ -0,0 +1,235 @@
+#include QMK_KEYBOARD_H
+
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+
+enum {
+  TD_CTRL_ESC
+};
+
+qk_tap_dance_action_t tap_dance_actions[] = {
+  [TD_CTRL_ESC]    = ACTION_TAP_DANCE_DOUBLE(KC_LCTL, KC_ESC)
+};
+
+#define _QWERTY 0
+#define _COLEMAK 1
+#define _DVORAK 2
+#define _LOWER 3
+#define _RAISE 4
+#define _ADJUST 16
```

Comment:
Just an FYI for future:

The better way to do this is to use an `enum` statement:

```c
enum layer_names {
    _QWERTY,
    _COLEMAK,
    _DVORAK,
    _LOWER,
    _RAISE,
    _ADJUST,
};
```

This defines each layer with a successive index starting from 0, so `_QWERTY` is `0`, `_COLEMAK` is `1`, and so on.

Assigning `_ADJUST` to `16` means the compiler creates arrays at each index that you skip (so 5-15), which bloats the installed size of the firmware, but with no difference in functionality.

This code:

```shell
Checking file size of keebio_nyquist_rev2_revkev.hex                                                [OK]
 * The firmware size is fine - 26380/28672 (92%, 2292 bytes free)
```

With an `enum`:

```shell
Checking file size of keebio_nyquist_rev2_revkev.hex                                                [OK]
 * The firmware size is fine - 25060/28672 (87%, 3612 bytes free)
```

Reply:
Agreed, thank you! The extra details are very cool.